tests/test_basic.py

```diff
@@ -1,17 +1,30 @@
 #!/usr/bin/env python
-# Copyright 2016-2021 XMOS LIMITED.
+# Copyright 2016-2024 XMOS LIMITED.
 # This Software is subject to the terms of the XMOS Public Licence: Version 1.
-import xmostest
+import pytest
+import utils
+import subprocess
+import logging
+from pathlib import Path
 
-def runtest():
-    testlevel = 'smoke'
-    resources = xmostest.request_resource('xsim')
+def test_version():
```

Comment:
I think you could remove this function. Then parametrize both using a top level list (see below)

targets = ["version" , "basic"]

Reply:
I did that in https://github.com/xmos/lib_device_control/pull/112/commits/9a4804325562b0be8fbd406400e1ff87caf961b9.